### DISCUSSION TITLE
Setting for hiding or showing the state of the target entity

### DISCUSSION THREAD

**Discussion by jesmak** [cgiesche/streamdeck-homeassistant#308] [Feature Requests]
I use this integration to have various toggles for lights etc. from HA in my Stream Deck. I updated the plugin today and was a bit bummed to see that the state of the target entity is now visible on the button as text. For toggles, it is very much redundant as you can already see the current state from the button color. For toggles, it just adds unnecessary clutter to the buttons. I get that it makes sense for various other scenarios, but would be nice to be able to show/hide it per button.

![image](https://github.com/user-attachments/assets/cf72a98b-eb7d-4900-b77f-f859ba8b83a1)

**Comment by cgiesche** [comment]
Hi,

you are totally right. Switches should not contain a default label. I will fix this soon(tm).
Until then, you can simply enable `Custom labels` and enter a single space character.
![grafik](https://github.com/user-attachments/assets/86e444fd-4779-46fb-ab94-00b41c80e603)

I hope this helps.

**Reply by jesmak** [reply]
Hey and thanks for the quick response! This workaround is good for me. However as it's not obvious how to hide the label, for future versions I would suggest having a selection between "show default label", "hide label" and "use custom label" to make it a bit more easier for new users.

**Comment by cgiesche** [comment]
I just fixed this in the default config. The workaround should not be necessary at least in about 7 days (unpredictable caching :/ )
